install/kubernetes/cilium/charts/config/templates/configmap.yaml

```
@@ -416,6 +416,9 @@ data:
   install-iptables-rules: {{ .Values.global.installIptablesRules | quote }}
 {{- if .Values.global.iptablesLockTimeout }}
   iptables-lock-timeout: {{ .Values.global.iptablesLockTimeout | quote }}
+{{- end }}
+{{- if .Values.global.ipTablesRandomFully }}
+  iptables-random-fully: {{ .Values.global.ipTablesRandomFully | quote }}
```

On the Helm side, although many of the current examples use `.Values.global`, ideally we should just use `.Values` here so that folks who upgrade are not forced to select a value for this global variable when attempting to upgrade from an earlier release:

```suggestion
{{- if .Values.ipTablesRandomFully }}
  iptables-random-fully: {{ .Values.ipTablesRandomFully | quote }}
```

---

Thanks, updated